tests/comfy_cli/command/test_code_search.py

```
@@ -120,23 +120,33 @@ def limit_hit_response(limit_hit_search):
     def test_with_repo_and_count(self):
         result = _build_query("LoadImage", "ComfyUI", 100)
-        assert result == "repo:^Comfy\\-Org/ComfyUI$ count:100 LoadImage"
+        assert result == "repo:^Comfy\\-Org/ComfyUI$ type:file count:100 LoadImage"
+
+    def test_user_type_filter_preserved(self):
+        """Don't inject type:file when the user already specified a type: filter."""
+        result = _build_query("type:commit fix bug", None, DEFAULT_COUNT)
+        assert "type:file" not in result
+        assert result == f"count:{DEFAULT_COUNT} type:commit fix bug"
+
+    def test_user_type_file_not_duplicated(self):
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)